packages/itwinui-react/src/core/Badge/Badge.tsx

```diff
@@ -45,6 +23,7 @@ export type BadgeProps = {
    */
   backgroundColor?:
     | 'primary'
+    | 'informational'
```

Comment:
I added `'informational'` as a possible option for `backgroundColor` since the data attribute doesn't recognize `'primary'` as a status option. I think we should replace `'primary'` with `'informational'` in our next major release